schemas/stsci.edu/asdf/transform/label_mapper-0.1.0.yaml

```diff
@@ -0,0 +1,138 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/label_mapper-0.1.0"
+tag: "tag:stsci.edu:asdf/transform/label_mapper-0.1.0"
+title: >
+  Represents a _LabelMapper.
+description: |
+  A label mapper instance maps locations to a label.
+  It is used together with RegionsSelector. The LabelMapper
```

Comment:
Here you use Python class names, but should probably use ASDF tag names.  You can refer to other tags in ASDF using markdown, e.g.:

```
It is used together with [regions_selector](ref:regions_selector)
```